### ISSUE TITLE
Enter Full Screen menu line repeated

### ISSUE THREAD

**Issue by minusf** [macvim-dev/macvim#566]
not really sure how to reproduce it, i just notice once in a while when i want to see the window list under the `Window` menu. maybe others have more details.

<img width="533" alt="screen shot 2017-10-09 at 11 05 58" src="https://user-images.githubusercontent.com/3632883/31820663-a5c9b602-b5a2-11e7-9518-9fa8dec75def.png">

```
Sierra 10.12.6

VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Oct 10 2017 11:48:35)
MacOS X (unix) version
Included patches: 1-1175
Compiled by Homebrew
```

**Comment by isaki-x** [comment]
I have seen this issue on High Sierra 10.13.1 as well:

```
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Nov  6 2017 02:39:16)
macOS version
Included patches: 1-1272
Compiled by root@Traviss-Mac-713.local
```

**Comment by bkmdev** [comment]
Same for me in OSX 10.13.1, @isaki-x and still seems to be broken in newer MacVim versions too 🙁 (see below for complete version and build info).  

Does anyone have a version installed where this did _not_ occur?  If so, perhaps we can use git's `bisect` to determine when this issue was first introduced, and thus the code responsible for it?  

```
:version
VIM - Vi IMproved 8.0 (2016 Sep 12, compiled Dec  7 2017 11:42:48)
macOS version
Included patches: 1-1376
Compiled by Homebrew
Huge version with MacVim GUI.  Features included (+) or not (-):
+acl               +clipboard         +digraphs          -footer            +libcall           -mouse_gpm         +odbeditor         +rightleft         +tcl               +virtualedit       -xpm
+arabic            +cmdline_compl     +dnd               +fork()            +linebreak         -mouse_jsbterm     +packages          +ruby              +termguicolors     +visual            -xsmp
+autocmd           +cmdline_hist      -ebcdic            +fullscreen        +lispindent        +mouse_netterm     +path_extra        +scrollbind        +terminal          +visualextra       -xterm_clipboard
-autoservername    +cmdline_info      +emacs_tags        -gettext           +listcmds          +mouse_sgr         +perl              +signs             +terminfo          +viminfo           -xterm_save
+balloon_eval      +comments          +eval              -hangul_input      +localmap          -mouse_sysmouse    +persistent_undo   +smartindent       +termresponse      +vreplace
+balloon_eval_term +conceal           +ex_extra          +iconv             -lua               +mouse_urxvt       +postscript        +startuptime       +textobjects       +wildignore
+browse            +cryptv            +extra_search      +insert_expand     +menu              +mouse_xterm       +printer           +statusline        +timers            +wildmenu
++builtin_terms    +cscope            +farsi             +job               +mksession         +multi_byte        +profile           -sun_workshop      +title             +windows
+byte_offset       +cursorbind        +file_in_path      +jumplist          +modify_fname      +multi_lang        +python            +syntax            +toolbar           +writebackup
+channel           +cursorshape       +find_in_path      +keymap            +mouse             -mzscheme          -python3           +tag_binary        +transparency      -X11
+cindent           +dialog_con_gui    +float             +lambda            +mouseshape        +netbeans_intg     +quickfix          +tag_old_static    +user_commands     -xfontset
+clientserver      +diff              +folding           +langmap           +mouse_dec         +num64             +reltime           -tag_any_white     +vertsplit         +xim
   system vimrc file: "$VIM/vimrc"
     user vimrc file: "$HOME/.vimrc"
 2nd user vimrc file: "~/.vim/vimrc"
      user exrc file: "$HOME/.exrc"
  system gvimrc file: "$VIM/gvimrc"
    user gvimrc file: "$HOME/.gvimrc"
2nd user gvimrc file: "~/.vim/gvimrc"
       defaults file: "$VIMRUNTIME/defaults.vim"
    system menu file: "$VIMRUNTIME/menu.vim"
  fall-back for $VIM: "/Applications/MacVim.app/Contents/Resources/vim"
Compilation: clang -c -I. -Iproto -DHAVE_CONFIG_H -DFEAT_GUI_MACVIM -Wall -Wno-unknown-pragmas -pipe  -DMACOS_X -DMACOS_X_DARWIN  -F/usr/local/opt/python/Frameworks -I/usr/local/Cellar/python/2.7.14/Frameworks/Python.framew
ork/Versions/2.7/include/python2.7 -I/usr/local/Cellar/python/2.7.14/Frameworks/Python.framework/Versions/2.7/include/python2.7 -fno-strict-aliasing -fno-common -dynamic -g -O2 -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototy
pes -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1
Linking: clang   -L.             -L /BuildRoot/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.Internal.sdk/usr/local/libressl/lib -L/BuildRoot/Applications/Xcode.app/Contents/
Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.Internal.sdk/usr/local/lib -L.             -L /BuildRoot/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.Internal
.sdk/usr/local/libressl/lib -L/BuildRoot/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.13.Internal.sdk/usr/local/lib -F/usr/local/opt/python/Frameworks -L/usr/local/opt/python/F
rameworks/Python.framework/Versions/2.7/lib/python2.7/config -lpython2.7 -framework CoreFoundation -L/usr/local/lib -o Vim -framework Cocoa -framework Carbon       -lm  -lncurses -liconv -framework AppKit   -fstack-protecto
r  -L/System/Library/Perl/5.18/darwin-thread-multi-2level/CORE -lperl -F/usr/local/opt/python/Frameworks -framework Python  -F/System/Library/Frameworks -framework Tcl -framework CoreFoundation -framework Ruby

```

**Comment by eirnym** [comment]
Does it still an issue? I can't reproduce it

**Comment by isaki** [comment]
This happened to me relatively recently on 10.14.4 (I just recently upgraded to 10.14.5; I'll if I can make it happen again).

**Comment by isaki** [comment]
I have not been able to reproduce on 10.14.5 with MacVim 8.1.950 (155).

**Comment by isaki** [comment]
I just reproduced it.

<img width="449" alt="image" src="https://user-images.githubusercontent.com/10616227/62892401-2e95a500-bd16-11e9-865d-a2979258e4a2.png">

**Comment by eirnym** [comment]
@isaki You're lucky! Could you share any way to reproduce it all the time?

**Comment by isaki** [comment]
Actually, I just figured that out a few minutes ago when I noticed it again and realized I had just double opened a file a few times (I had a find command opening some files for me, some of which were already open). However, testing now, I've come up with an even simpler method.

I'm using gvim on the command line (which is just `/Applications/MacVim.app/Contents/bin/gvim`). Just do this:

```
gvim foo
```

Do not not quit it. It doesn't have to be a real file. Just every time you get the warning hit abort. The first time nothing bad happens. Do it again. And again. You will get (n - 1) Enter Full Screen entries where n is the number of aborts.

<img width="275" alt="image" src="https://user-images.githubusercontent.com/10616227/63053167-24f37500-beaf-11e9-96d9-b36fb8d20f75.png">
